target/step159-2020-1/script.js

```diff
@@ -0,0 +1,27 @@
+function loadBody() {
```

Comment:
This looks like duplicate code? Do you mean to have two copies of the same code?

Reply:
Sorry, where is this duplicated?

Reply:
This used to look fairly identical to the code in src/main/webapp/script.js

What's the purpose of the changes in this file?

Reply:
I had to push my current code to restructure our filesystem a few days ago - maybe that's what you mean? But since I didn't get that reviewed, I wanted to commit it now to make sure everything looked good with it before formally using it.

Reply:
So which code path is actually being used? These files in `target/step159-2020-1` or the files above in `src/main/webapp`?

Reply:
I put the target folder in .gitignore, not sure why it's still showing up. I think all of us are having an issue with those files being duplicated in target even though we're working in src/main/webapp